gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/apim/core/portal_page/use_case/SetHomepageUseCaseTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.apim.core.portal_page.use_case;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import inmemory.PortalPageCrudServiceInMemory;
+import io.gravitee.apim.core.portal_page.model.GraviteeMarkdown;
+import io.gravitee.apim.core.portal_page.model.PageLocator;
+import io.gravitee.apim.core.portal_page.model.PortalPage;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class SetHomepageUseCaseTest {
+
+    private PortalPageCrudServiceInMemory crudService;
+    private SetHomepageUseCase useCase;
+
+    @BeforeEach
+    void setUp() {
+        crudService = new PortalPageCrudServiceInMemory();
+        useCase = new SetHomepageUseCase(crudService);
+    }
+
+    @Test
+    void shouldSetHomepageWhenPageExists() {
```

Comment:
It's just a detail but I think it would be great to agree on test naming conventions 🤔 I have seen some tests using `_`and `@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)` some others using camelcase in the backend